operator/internal/handlers/internal/storage/secrets.go

```diff
@@ -154,57 +150,55 @@ func extractS3SSEConfig(d map[string][]byte) (storage.S3SSEConfig, error) {
 
 func extractSwiftConfigSecret(s *corev1.Secret) (*storage.SwiftStorageConfig, error) {
 	// Extract and validate mandatory fields
-	url := s.Data["auth_url"]
+	url := s.Data[storage.KeyOSSwiftAuthURL]
```

Comment:
Does it make sense to reduce clutter and remove "OS" from all the Swift constants?